equinix/fabric_connection_read_schema.go

```diff
@@ -419,10 +419,10 @@ func readConnectionSideAccessPointSch() map[string]*schema.Schema {
 				Schema: readServiceProfileSch(),
 			},
 		},
-		"gateway": {
+		"router": {
```

Comment:
This is a breaking change; the process laid out here will enable a smoother upgrade for users: https://developer.hashicorp.com/terraform/plugin/sdkv2/best-practices/deprecations#renaming-a-computed-attribute

Reply:
This is now fixed with the latest commit. Please check and approve.